### ISSUE TITLE
Unsign return signature breaking change in 4.x release line

### ISSUE THREAD

**Issue by mmarti** [fastify/fastify-cookie#103]
## 💥 Regression Report

Prior to the 4.2.0 release, `reply.unsignCookie(...)` would return a string.  After the PR, it now returns an object.  It looks like previously you were using `cookie-signature`'s logic directly, but with the custom signer, you started wrapping and returning your own object.

Breaking commit: https://github.com/fastify/fastify-cookie/compare/v4.2.0...master#diff-da0818d1007a8f324a63ef085c6b9bbd765b262a841385b2e7b0f28f230ce349R29

## Last working version

Worked up to version: 4.1.1

Stopped working in version: 4.2.0

## To Reproduce

Try unsigning a cookie.

```js
reply.unsignCookie(req.cookies.mycoolcookie);
// returns string in 4.1.1
// returns object in 4.2.0
```

## Expected behavior

It's unclear whether this was an intentional breaking change or if there was a misunderstanding of the existing signature.  Ultimately, with 4.2.0 you need to grab the `value` property of the unsigned cookie object to restore similar/old behavior, but I wanted to confirm if this was correct/expected behavior.

I created this issue more so to clarify, as I was confused when bumping versions and had things start breaking.

## Your Environment

- *node version*: 14.15.4
- *fastify version*: >=3.11.0
- *os*: Linux

**Comment by mmarti** [comment]
Actually, I think I might have gotten the commits messed up; perhaps it was indeed a breaking change included in the 5.x release.  If that's the case, this can likely be closed, but is it possible to get a CHANGELOG or something added in the future for changes like this, or perhaps there already is a place and I'm just missing it.  Thanks.

**Comment by mcollina** [comment]
That change was shipped as v5.0.0, as it was a breaking change.. and the reason why we bumped major.
